repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/released-upcoming-publication.yaml

```diff
@@ -0,0 +1,75 @@
+---
+
+/content/documents/corporate-website/publication-system/released-upcoming-publication:
```

Comment:
Should really rename to be consistent with our standardisation on using the hippo term "published", rather than "released"

Reply:
Done.